runner/ollamarunner/runner.go

```diff
@@ -687,7 +687,7 @@ func (s *Server) loadModel(
 	multiUserCache bool,
 ) {
 	var err error
-	s.model, err = model.New(mpath, params)
+	s.model, err = model.New(context.TODO(), mpath, params)
```

Comment:
We actually do have a context in the runner already, maybe we could just pass that one in?
https://github.com/ollama/ollama/blob/fb6252d786c9cafdd2dbfa434535d85c611c0ff0/runner/ollamarunner/runner.go#L801